### ISSUE TITLE
Running past migration after init with shared context

### ISSUE THREAD

**Issue by pumazi** [karenc/db-migrator#10]
I've run into an error on 'migrate' after initializing a fresh database.

The following steps should reproduce the error:

1. dropdb, if you've got one
1. createdb
1. cnx-archive-initdb & cnx-publishing-initdb
1. dbmigrator --config app.ini --context cnx-archive init
1. dbmigrator --config app.ini --context cnx-publishing migrate

```pytb
Traceback (most recent call last):
  File "/var/cnx/venvs/publishing/bin/dbmigrator", line 9, in <module>
    load_entry_point('db-migrator==0.0.7', 'console_scripts', 'dbmigrator')()
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/dbmigrator/cli.py", line 70, in main
    return args['cmmd'](**args)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/dbmigrator/utils.py", line 66, in wrapper
    return func(cursor, *args, **kwargs)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/dbmigrator/commands/migrate.py", line 30, in cli_command
    migration)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/dbmigrator/utils.py", line 121, in compare_schema
    callback(*args, **kwargs)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/dbmigrator/utils.py", line 132, in run_migration
    migration.up(cursor)
  File "../../var/cnx/venvs/publishing/local/lib/python2.7/site-packages/cnxpublishing/sql/migrations/20160104115058_add_api_keys_table.py", line 18, in up
    """)
psycopg2.ProgrammingError: relation "api_keys" already exists
```

Also, the schema_migrations database looks like this:

```
cnxarchive=# select * from schema_migrations;
 version | applied 
---------+---------
(0 rows)
```

**Comment by pumazi** [comment]
As we discussed a few hours ago, 'init' enters existing migrations into the schema_migrations table. The problem is that our 'init' never finds the migrations in cnx-publishing, because this package is not currently in context. :frowning:

**Comment by karenc** [comment]
So... there are 2 things to fix here:

 - `init` shouldn't mess with the data in the schema_migrations table if it already exists,
 - there needs to be a way to specify multiple contexts like `--context=cnx-archive,cnx-publishing`.

**Comment by pumazi** [comment]
Hopefully resolved via #11
